recipes/festim/meta.yaml

```diff
@@ -30,6 +30,7 @@ test:
   imports:
     - festim
   commands:
+    - pip list
```

Comment:
You don't need that and its results are not reliable in a conda environment.

```suggestion
```

Reply:
Thanks! Was trying to figure out why the build was failing